ASPNetCoreMastersTodoList.API/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using ASPNetCoreMastersTodoList.API.BindingModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Services;
+using Services.DTO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ASPNetCoreMastersTodoList.API.Controllers
+{
+    public class ItemsController : ControllerBase
+    {
+        private readonly ItemService _itemService;
+
+        public ItemsController()
+        {
+            this._itemService = new ItemService();
+        }
+
+        [HttpGet]
+        public int Get(int Id)
+        {
+            return _itemService.GetAll(Id);
+        }
+
+        [HttpPost]
+        public void Post([FromQuery] ItemCreateBindingModel model)
```

Comment:
This is not a requirement however I think changing binding model source to [FromBody] will be more appropriate for the use case. Thanks!